lib/connectors/memory.js

```diff
@@ -3,6 +3,7 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+var g = require('strong-globalize')();
```

Comment:
I'm thinking this is probably better in the long run since its a one liner versus two. @0candy Thoughts?